chia/pools/pool_wallet.py

```diff
@@ -229,16 +229,17 @@ async def get_unconfirmed_transactions(self) -> List[TransactionRecord]:
     async def get_tip(self) -> Tuple[uint32, CoinSpend]:
         return self.wallet_state_manager.pool_store.get_spends_for_wallet(self.wallet_id)[-1]
 
-    async def update_pool_config(self, make_new_authentication_key: bool):
+    async def update_pool_config(self) -> None:
         current_state: PoolWalletInfo = await self.get_current_state()
         pool_config_list: List[PoolWalletConfig] = load_pool_config(self.wallet_state_manager.root_path)
         pool_config_dict: Dict[bytes32, PoolWalletConfig] = {c.launcher_id: c for c in pool_config_list}
         existing_config: Optional[PoolWalletConfig] = pool_config_dict.get(current_state.launcher_id, None)
+        payout_instructions: str = existing_config.payout_instructions if existing_config is not None else ""
 
-        if make_new_authentication_key or existing_config is None:
-            payout_instructions: str = (await self.standard_wallet.get_new_puzzlehash(in_transaction=True)).hex()
-        else:
-            payout_instructions = existing_config.payout_instructions
+        if len(payout_instructions) == 0:
+            self.log.info("New config entry, generating new puzzle hash...")
+            payout_instructions = (await self.standard_wallet.get_new_puzzlehash(in_transaction=True)).hex()
+            self.log.info(f"Generated payout_instructions puzzle hash: {payout_instructions}")
```

Comment:
I'd probably prefer this all on one line in the log

Reply:
![image](https://user-images.githubusercontent.com/339312/156281165-8e49eb4a-c5c7-4ae3-bac2-2b8fc1f53159.png)

Reply:
![As you wish](https://media.giphy.com/media/rmAxpa2ZCnRYs/giphy.gif)